api/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
     runtimeOnly 'com.h2database:h2'
     annotationProcessor 'org.projectlombok:lombok'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation 'org.mockito:mockito-core:4.6.1'
```

Comment:
This line is unnecessary, 'org.springframework.boot:spring-boot-starter-test' has included it.